local-volume/utils/update-pv-to-beta/deployment/kubernetes/admin-account.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: local-storage-update
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: local-storage:updater
+subjects:
+- kind: ServiceAccount
+  name: local-storage-update
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
This is pretty extensive permissions.  We just need PV read and update permissions right?

Reply:
use `system:persistent-volume-provisioner` instead.
Should we create a new ClusterRole ?